config/sync/core.date_format.article_author_date.yml

```diff
@@ -0,0 +1,8 @@
+uuid: cbff7fb1-23be-469b-a2c1-ae834ec4c55e
+langcode: en
+status: true
+dependencies: {  }
+id: article_author_date
+label: 'Article author date'
+locked: false
+pattern: 'd. F y'
```

Comment:
Consider if it is a good idea to provide a date format with a very specific name which implies a specific use case.

In the design do we see dates with the same granularity (day, month, year) presented differently in other contexts. If so then that might imply yes. If not then we should probably create something with a name that invites reuse.

If in doubt here then talk to a colleague.